src/legacy/ui/public/kbn_top_nav/kbn_top_nav.js

```diff
@@ -24,7 +24,7 @@ import { TopNavMenu } from '../../../core_plugins/kibana_react/public';
 
 const module = uiModules.get('kibana');
 
-module.directive('kbnTopNav', () => {
+export function createTopNavDirective() {
```

Comment:
exporting the directive definition as a module export to be able to put it in a separate angular module with a different name. This is done to use existing code like this one but make sure it's not using any dependencies from the global angular instance